go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go

```diff
@@ -28,6 +28,7 @@ import (
 
 func init() {
 	sqle.MinRowsPerPartition = 2
+	json.FeatureFlag = true
```

Comment:
If the default value is true and there is no way to set it to false it may be time to kill the flag.  That or default it to false and require --json

Reply:
It defaults to false [here](https://github.com/dolthub/dolt/blob/61356c11fbce93d1d225588f6ac4fa9bcc3b6f68/go/libraries/doltcore/sqle/json/noms_json_value.go#L27) and there is a BATS test to assert that it's not available to users.
It only gets set true here for the duration of these unit tests.